Polygon_mesh_processing/test/Polygon_mesh_processing/test_orient_cc.cpp

```diff
@@ -0,0 +1,129 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Surface_mesh.h>
+#include <CGAL/Polygon_mesh_processing/orientation.h>
+#include <CGAL/Polygon_mesh_processing/corefinement.h>
+
+#include <iostream>
+#include <fstream>
+
+namespace PMP = CGAL::Polygon_mesh_processing;
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel Kernel;
+typedef CGAL::Surface_mesh<Kernel::Point_3> SMesh;
+
+template<class TriangleMesh, class NamedParameters>
+bool test_orientation(TriangleMesh& tm, bool is_positive, const NamedParameters& np)
+{
+  typedef boost::graph_traits<TriangleMesh> Graph_traits;
+  typedef typename Graph_traits::vertex_descriptor vertex_descriptor;
+  typedef typename Graph_traits::face_descriptor face_descriptor;
+  typedef typename CGAL::GetVertexPointMap<TriangleMesh,
+      NamedParameters>::const_type Vpm;
+  typedef typename CGAL::GetFaceIndexMap<TriangleMesh,
+      NamedParameters>::const_type Fid_map;
+
+  Vpm vpm = boost::choose_param(get_param(np, CGAL::internal_np::vertex_point),
+                                CGAL::get_const_property_map(boost::vertex_point, tm));
+
+  Fid_map fid_map = boost::choose_param(get_param(np, CGAL::internal_np::face_index),
+                                        CGAL::get_const_property_map(boost::face_index, tm));
```

Comment:
here `get_const_property_map()` is in the namespace `CGAL`.
To avoid confusion (and maybe some bugs, later), it should be made clear in all the patch what is in `boost::` and what is in `CGAL::`

Reply:
We are in the test that is not in the CGAL namespace, so CGAL:: is needed.

Reply:
I understand, but what about `get_param()`?
Reading the code, I was confused what is from CGAL and what is from boost. I think it should be made explicit everywhere

Reply:
I added the missing using.